test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/service-account-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: service-account-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Figured this too. The `system:kube-controller-manager` user is bound with the right role as part of the bootstrap policy bindings